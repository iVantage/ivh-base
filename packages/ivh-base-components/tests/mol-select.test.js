import React from 'react'
import renderer from 'react-test-renderer'
import {
  AuSelect,
  AuSelectButton,
  AuSelectMenu,
  AuSelectMenuItem
} from '../dist/ivh-base-components'

test.only('AuSelect should render consistently', () => {
  const tree = renderer
    .create(
      <AuSelect>
        <AuSelectButton>
          <button onClick={() => {}}>Toggle</button>
        </AuSelectButton>
        <AuSelectMenu>
          <AuSelectMenuItem onClick={() => {}}>Clicky 1</AuSelectMenuItem>
          <AuSelectMenuItem.Link href='https://example.com'>
            Clicky 2
          </AuSelectMenuItem.Link>
        </AuSelectMenu>
      </AuSelect>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
