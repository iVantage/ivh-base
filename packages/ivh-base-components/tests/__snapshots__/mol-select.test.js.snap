// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuSelect should render consistently 1`] = `
<div
  className="AuSelect au-dropdown"
  onBlur={[Function]}
  onKeyDown={[Function]}
  style={Object {}}
>
  <div
    className="au-dropdown-button"
    onClick={[Function]}
    onFocus={[Function]}
    style={
      Object {
        "display": "inherit",
      }
    }
  >
    <button
      onClick={[Function]}
    >
      Toggle
    </button>
  </div>
</div>
`;
